package.json

```diff
@@ -280,6 +280,86 @@
                   "type": "string"
                 },
                 "default": []
+              },
+              "pipeTransport": {
+                "type": "object",
+                "description": "Describes options to launch a program through a pipe transport.",
+                "default": {
+                  "pipeProgram": "enter pipe program name, for example plink.exe",
+                  "pipeArgs": []
+                },
+                "properties" : {
+                  "pipeProgram": {
```

Comment:
Does the MIEngine search PATH if this is not a full file path? Or does it need to be a file path? Probably worth calling out in the description either way.